### PR TITLE
Display removed machines in the changelog

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3113.

### CODE REVIEW

**Comment by jujugui** [comment]
Build finished.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8241/

File: jujugui/static/gui/src/app/components/deployment-flow/services/services.js [hunk 1]
```diff
@@ -52,30 +53,73 @@ class DeploymentServices extends React.Component {
     );
   }
 
-  render() {
-    const props = this.props;
-    const currentChangeSet = props.getCurrentChangeSet();
-    const changes = props.sortDescriptionsByApplication(
+  /**
+    Generate the removed machines if there are any.
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/services/services.js]: This description doesn't match :)

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**